paddle/fluid/pybind/pybind.cc

```diff
@@ -146,7 +146,7 @@ bool IsCompiledWithMKLDNN() {
 #endif
 }
 
-bool SupportsBfloat16() {
+bool SupportsBfloat16Int8() {
```

Comment:
I wonder if it is better to name `SupportsAvx512Core` and in comment add information that needed for bfloat16 and int8 evaluation. But your proposal is understandable too.

Reply:
I may change it. And since avx512_core is defined only in Paddle
```
case avx512_core:
      return true && cpu.has(Cpu::tAVX512F) && cpu.has(Cpu::tAVX512BW) &&
             cpu.has(Cpu::tAVX512VL) && cpu.has(Cpu::tAVX512DQ);
```
I will add the needed flag in message too.
```
@unittest.skipIf(not core.supports_avx512_core(),
                 "Place does not support oneDNN BF16 or INT8 because following flags are missing: avx512f avx512dq avx512bw avx512vl")
```

Since CI is very slow, I wait this commit to pass all CIs and then I will commit a new one. Thanks for your suggesiton ~